dags/sentinel-2_indexing.py

```diff
@@ -68,30 +94,38 @@
         name="datacube-index",
         task_id="indexing-task",
         get_logs=True,
+        is_delete_operator_pod=True,
     )
 
-    # TODO: implement
     UPDATE_RANGES = KubernetesPodOperator(
         namespace="processing",
         image=OWS_IMAGE,
         cmds=["datacube-ows-update"],
```

Comment:
need to run the --views first, it's a 2 step operation to update ranges now

1 `datacube-ows-update --views --blocking` updates the sql views
2 `datacube-ows-update s2_l2a` updates the product